test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -209,31 +209,48 @@ TEST(validator_test, output_file)
     { // single file
 
         { // empty list of file.
-            seqan3::output_file_validator my_validator{};
+            seqan3::output_file_validator my_validator{seqan3::output_file_open_mode::exclusive};
             EXPECT_NO_THROW(my_validator(tmp_name.get_path()));
         }
 
-        { // file does not exist.
+        { // file does exist but allow to overwrite it (without given formats to the validator)
             std::ofstream tmp_file_2(tmp_name_2.get_path());
             std::filesystem::path does_not_exist{tmp_name_2.get_path()};
```

Comment:
Adapt the naming here, please:
```suggestion
            std::filesystem::path existing_path{tmp_name_2.get_path()};
```